release/README.md

```diff
@@ -0,0 +1,15 @@
+## EKS Anywhere Curated Packages Release Runbook
+
+The steps to create a new release of EKS-A Package Controller
+
+1. Tag the repository for example `git tag v0.2.2`
+1. Push the tag `git push :v0.2.2`
```

Comment:
This command is not working for me. 
I usually do `git push upstream v0.2.2`

Reply:
fixed